src/main/kotlin/com/team4099/robot2022/subsystems/Wheel.kt

```diff
@@ -148,32 +149,29 @@ class Wheel(
         "Drivetrain Tuning",
         "$label Azimuth kP",
         { Constants.Drivetrain.PID.DIRECTION_KP },
-        { newP -> directionPID.p = newP },
+        { newP -> directionFalcon.config_kP(0, newP) },
         false)
 
-    directionPID.p = Constants.Drivetrain.PID.DIRECTION_KP
-    directionPID.i = Constants.Drivetrain.PID.DIRECTION_KI
-    directionPID.iZone = 0.0
-    directionPID.d = Constants.Drivetrain.PID.DIRECTION_KD
-    directionPID.ff = Constants.Drivetrain.PID.DIRECTION_KFF
-    directionPID.setSmartMotionMaxVelocity(
-        directionSensor.velocityToRawUnits(Constants.Drivetrain.DIRECTION_VEL_MAX), 0)
-    directionPID.setSmartMotionMaxAccel(
-        directionSensor.accelerationToRawUnits(Constants.Drivetrain.DIRECTION_ACCEL_MAX), 0)
-    directionPID.setOutputRange(-1.0, 1.0)
-    directionPID.setSmartMotionMinOutputVelocity(0.0, 0)
-    directionPID.setSmartMotionAllowedClosedLoopError(
-        directionSensor.positionToRawUnits(Constants.Drivetrain.ALLOWED_ANGLE_ERROR), 0)
-    directionSpark.setSmartCurrentLimit(Constants.Drivetrain.DIRECTION_SMART_CURRENT_LIMIT)
-
-    directionSpark.burnFlash()
-
-    drivePID.p = Constants.Drivetrain.PID.DRIVE_KP
-    drivePID.i = Constants.Drivetrain.PID.DRIVE_KI
-    drivePID.d = Constants.Drivetrain.PID.DRIVE_KD
-    drivePID.ff = Constants.Drivetrain.PID.DRIVE_KFF
-    driveSpark.setSmartCurrentLimit(Constants.Drivetrain.DRIVE_SMART_CURRENT_LIMIT)
-    driveSpark.burnFlash()
+    directionFalcon.config_kP(0, Constants.Drivetrain.PID.DIRECTION_KP)
+    directionFalcon.config_kI(0, Constants.Drivetrain.PID.DIRECTION_KI)
+    // directionPID.iZone = 0.0
+    directionFalcon.config_kD(0, Constants.Drivetrain.PID.DIRECTION_KD)
+    directionFalcon.config_kF(0, Constants.Drivetrain.PID.DIRECTION_KFF)
+    directionFalcon.configMotionCruiseVelocity(
+        directionSensor.velocityToRawUnits(Constants.Drivetrain.DIRECTION_VEL_MAX))
+    directionFalcon.configMotionAcceleration(
+        directionSensor.accelerationToRawUnits(Constants.Drivetrain.DIRECTION_ACCEL_MAX))
+    directionFalcon.configPeakOutputForward(1.0)
+    directionFalcon.configPeakOutputReverse(-1.0)
+    directionFalcon.configAllowableClosedloopError(
+        0, directionSensor.positionToRawUnits(Constants.Drivetrain.ALLOWED_ANGLE_ERROR))
+    // directionFalcon.outputCurrent(Constants.Drivetrain.DIRECTION_SMART_CURRENT_LIMIT)
+
+    driveFalcon.config_kP(0, Constants.Drivetrain.PID.DRIVE_KP)
+    driveFalcon.config_kI(0, Constants.Drivetrain.PID.DRIVE_KI)
+    driveFalcon.config_kD(0, Constants.Drivetrain.PID.DRIVE_KD)
+    driveFalcon.config_kF(0, Constants.Drivetrain.PID.DRIVE_KFF)
+    // driveFalcon.setSmartCurrentLimit(Constants.Drivetrain.DRIVE_SMART_CURRENT_LIMIT)
```

Comment:
ok wait are we not setting current limits at all rn? pls fix, there's definitely an equivalent method (though i think it sets supply side current limits so our numbers should be reduced from what they are for the sparks)